### PR TITLE
Add annotations for process-image relation

### PR DESCRIPTION
**Pull request by alexwatcher**
[x] parse annotation  integrity-monitor.scnsoft.com/process-image
[x] add argument to summator
[x] test added

### CODE REVIEW

File: pkg/mutate/mutate.go [hunk 1]
```diff
@@ -17,14 +19,31 @@ import (
...
 )
 
+var (
+	ErrInvalidProcessImageFormat = errors.New("invalid process-image format")
+
+	processImageRegexp *regexp.Regexp
+)
+
+func init() {
+	regexp, err := regexp.Compile(processImageRegexpStr)
```
Comment (alavrovinfb) [inline pkg/mutate/mutate.go]: Probably it is better to use `regexp.MustCompile` it panics on error

File: pkg/mutate/mutate.go [hunk 2]
```diff
@@ -108,3 +137,13 @@ func checkPathsAnnotations(annotations map[string]string) (found bool) {
 
 	return found
 }
+
+func checkProcessImage(annotations map[string]string) (bool, error) {
+	if annocation, ok := annotations[AnnotationProcessImage]; ok {
+		if !processImageRegexp.MatchString(annocation) {
```
Comment (alavrovinfb) [inline pkg/mutate/mutate.go]: seems to be a typo `annocation`

**Review by alavrovinfb — APPROVED** [review]

File: pkg/mutate/mutate.go (hunk 2 above)
Comment (alexwatcher) [reply pkg/mutate/mutate.go]: fixed

File: pkg/mutate/mutate.go (hunk 1 above)
Comment (alexwatcher) [reply pkg/mutate/mutate.go]: fixed

**Pull request merged**